modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTSchemaVersionLocalServiceImpl.java

```diff
@@ -119,13 +125,31 @@ public boolean isLatestCTSchemaVersion(long ctSchemaVersionId) {
 		CTSchemaVersion ctSchemaVersion =
 			ctSchemaVersionPersistence.fetchByPrimaryKey(ctSchemaVersionId);
 
-		if (ctSchemaVersion == null) {
+		try {
+			CTConflictConfiguration ctConflictConfiguration =
```

Comment:
shouldn't this check happen in isLatestCTSchemaVersion(long ctSchemaVersionId, boolean strict)?